news/2018-06-30-cwc-2018-recap-quarter-finals.md

```diff
@@ -0,0 +1,51 @@
+
+Unfortunately, we saw our first two “win by default” results of the tournament this weekend. Sweden and Finland forfeited their Losers' Bracket matches to Poland and Mexico respectively. This eliminated the two Scandinavian teams from the tournament. Hopefully we will see them again next year.
+
+Next up, Netherlands clashed with Japan – both teams were at risk of being eliminated and sent home, so they put their all on the line. Netherlands bounced back strongly from last week’s loss, taking an early lead and never looking back. Japan was unable to maintain consistency with the Dutch team and would eventually fall 5-1.
+
+Then came the match everyone was waiting for. China. The United States. So much at stake for both teams. After all the amazing moments and matches we've had so far in this tournament, it was hard to imagine anything topping what we had already seen. But leave it to the top two teams to shock us all!
+
+The match had no break points – both teams would win all of their picks which meant one thing, and one thing only. Tiebreaker. It's the kind of situation that you dream of when you're a kid. Bottom of the ninth, two outs, bases loaded, down by three. Who would be the hero?
+
+![](/wiki/shared/news/2018-06-30-cwc-2018-recap-quarter-finals/CNvUS_game9_720w.jpg)
+
+Not one, not two, but three players stepped up to the platter. Guillotine and Playboy put in full combos for the United States and Dusk would put up an SS for China. But the SS wouldn't be enough. The Americans pulled off the impossible by beating China to advance to Finals Week in the first ever encounter between the two goliaths in the osu!catch World Cup. China were sent to the Losers' Bracket in the Quarter Finals. Yeah I know, crazy right? I was there.
+
+Tasked with following up one of the most hyped matches of all time, Chile and Taiwan would also do battle with a trip to Finals Week on the line. Unfortunately, Taiwan were never able to establish substantial momentum in the early stages of the match. Although both teams played well, Chile would come out on top 5-1, advancing to Finals Week once again in prime position to make a bid for a top 3 finish.
+
+The closing match saw France and Norway square off in the Losers' Bracket. France continued to impress, routing the Norwegians 5-0. Norway played well, but were simply outmatched by the French. It has been an excellent year for the Norwegians however, and we will definitely be looking forward to more from them in the future.
+
+## Upcoming Stage - Semi Finals
+
+The hype only gets bigger as we transition into what could be a historic week of matches for some of our remaining teams. In the Winners' Bracket, The United States are looking to clinch their first ever podium spot, whilst Chile are looking to return the podium for the first time since they won it all in 2013. Both will face fierce competition from the ever-consistent Indonesia and a growing-in-strength South Korea respectively.
```

Comment:
`Bracket, The United States are` -> `Bracket, the United States is`

Reply:
The "the" is incorrect and has been fixed for consistency, but since US is technically a plural, "are" makes more sense to read.